diff -uNr apache2-orig/ports.conf apache2/ports.conf
--- apache2-orig/ports.conf	2009-11-13 07:48:49.000000000 +0900
+++ apache2/ports.conf	2009-12-02 07:55:11.033316073 +0900
@@ -5,8 +5,8 @@
 # Debian etch). See /usr/share/doc/apache2.2-common/NEWS.Debian.gz and
 # README.Debian.gz
 
-NameVirtualHost *:80
-Listen 80
+NameVirtualHost *:6244
+Listen 6244
 
 <IfModule mod_ssl.c>
     # SSL name based virtual hosts are not yet supported, therefore no
diff -uNr apache2-orig/sites-available/default apache2/sites-available/default
--- apache2-orig/sites-available/default	2009-12-02 07:56:11.943316029 +0900
+++ apache2/sites-available/default	2009-12-02 07:48:25.943316031 +0900
@@ -1,4 +1,4 @@
-<VirtualHost *:80>
+<VirtualHost *:6244>
 	ServerAdmin webmaster@localhost
 
 	DocumentRoot /var/www
@@ -38,4 +38,19 @@
         Allow from 127.0.0.0/255.0.0.0 ::1/128
     </Directory>
 
+    #### for hyeoncheon
+    <Location "/">
+      Order deny,allow
+      Deny from all
+      Allow from 10.250.0.0/16 10.200.84.0/23 203.235.222.0/23
+    </Location>
+
+    FastCGIExternalServer /var/www/dummyname.fcgi -host 127.0.0.1:3001
+    Alias /media /opt/hyeoncheon/media
+    RewriteEngine On
+    RewriteRule ^/(media.*)$ /$1 [QSA,L,PT]
+    RewriteCond %{REQUEST_FILENAME} !-f
+    RewriteRule ^/(.*)$ /dummyname.fcgi/$1 [QSA,L]
+    #### for ongsung end
+
 </VirtualHost>
